packages/api/src/controllers/asset.test.ts

```diff
@@ -316,5 +324,58 @@ describe("controllers/asset", () => {
         },
       });
     });
+    describe("chunked upload", () => {
+      jest.setTimeout(10000);
+      it("should start upload, stop it, resume it on tus test server", async () => {
+        const filename = "test.mp4";
+        const path = "/tmp";
+        const filePath = `${path}/${filename}`;
+        const res = await client.post("/asset/request-upload", {
+          name: "tus-test",
+        });
+        await createMockFile(filePath, 1024 * 1024 * 10);
+        const file = fs.createReadStream(filePath);
+        const { size } = fs.statSync(filePath);
+        expect(res.status).toBe(200);
+        let { tusEndpoint } = await res.json();
+        tusEndpoint = tusEndpoint.split("/api/")[1];
+        tusEndpoint = `${client.server.host}/api/${tusEndpoint}`;
+        await new Promise<void>((resolve, reject) => {
```

Comment:
You are not really performing any tests from the process performed "inside" this promise apart from ensuring it doesn't throw any exceptions. Some things missing: 
 - Actually interrupting the upload and then starting a new one to resume it
 - Finishing the upload and checking if the file is now available in the `/tmp` directory (or wherever they are uploaded)